pkg/template/release.go

```diff
@@ -59,13 +59,13 @@ func ExtractComponentVersions(releaseComponents []v1alpha1.ReleaseSpecComponent)
 	return versions, nil
 }
 
-func buildImage(registryDomain, repo, tag string) string {
+func buildImage(registryDomain, repo, tag, suffix string) string {
 	name := repo
 	if registryDomain != "" {
 		name = registryDomain + "/" + repo
 	}
 
-	return name + ":" + tag
```

Comment:
Paweł just changed this from `fmt.Sprintf` to direct string appending I think. Should we prefer one or the other?

Reply:
I am a fan of `fmt` - just committed my change.